newrelic-agent/src/main/java/com/newrelic/agent/tracers/AbstractTracer.java

```diff
@@ -50,6 +50,7 @@
     private String customPrefix = "Custom";
     // doesn't need to be thread safe since this flag affects the decision to registerAsync
     private Boolean trackChildThreads = null;
+    private Boolean trackCallBackRunnable = false;
```

Comment:
`AtomicBoolean` here to help with threading issues?

Reply:
I don't think this needs to be thread safe as the tracer/TransactionActivity are thread specific